src/Avalonia.Base/Input/GestureRecognizers/PinchGestureRecognizer.cs

```diff
@@ -13,6 +13,8 @@ public class PinchGestureRecognizer : StyledElement, IGestureRecognizer
         private Point _secondPoint;
         private Point _origin;
 
+        public IInputElement? Target => _target;
```

Comment:
Why do we need this property to be public?

Reply:
It's used as Source element when the Pointer device raises Pointer events on the GestureRecognizer